src/index.html

```diff
@@ -2,15 +2,49 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="www.mateacademy.com"
+    metod="post"
+    class="form-large"
+    data-qa="big"
+    >
+      <label for="large-form">
+        <input
+        id="large-form"
+        class="large"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
```

Comment:
```suggestion
        <input
          id="large-form"
          class="large"
          data-qa="keypress"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
```
У атрибутов должен быть отступ